Userland/Utilities/xargs.cpp

```diff
@@ -44,10 +44,10 @@ ErrorOr<int> serenity_main(Main::Arguments main_arguments)
 
     StringView placeholder;
     bool split_with_nulls = false;
-    char const* specified_delimiter = "\n";
+    DeprecatedString specified_delimiter = "\n"sv;
     Vector<DeprecatedString> arguments;
     bool verbose = false;
-    char const* file_to_read = "-";
+    DeprecatedString file_to_read = "-"sv;
```

Comment:
Why are these not StringViews?

Reply:
Because they're used later on in C APIs.

Reply:
huh I thought i checked, consider this comment null and void